src/.github/workflows/{% if ci == 'GitHub' %}test.yml{% endif %}.jinja

```diff
@@ -12,6 +12,7 @@ on:
 {%
 set IMAGES = {
   "odoo": {
+    18.0: "ghcr.io/oca/oca-ci/py3.10-odoo18.0:latest",
```

Comment:
python 3.10 here doesn't match python-version 3.12 above?

Reply:
3.10 is correct, per
https://github.com/odoo/odoo/blob/9e35989436b807d5db738116cd3982dcb4efe81c/setup.py#L66